complex/ft_comp.c

```diff
@@ -54,8 +54,6 @@ static int ft_open_cntrs(void)
 {
 	int ret;
 
-	opts.comp_method = test_info.cq_wait_obj;
-
```

Comment:
I didn't follow this.  opts.comp_method is typically set directly by the test in question to indicate how it wants to process completions.   The value needs to be set prior to opening the counters, since the counter attributes are selected based on the value.

Having a function that opens counters set this and based on a CQ (not counter) attribute that was assigned based on the value is really confusing as to the intent.